tests/foreman/api/test_repository.py

```diff
@@ -671,11 +710,14 @@ def test_positive_update_checksum(self):
 
         :CaseImportance: Critical
         """
-        repo = entities.Repository(
-            product=self.product, checksum_type='sha1').create()
-        repo.checksum_type = 'sha256'
-        repo = repo.update(['checksum_type'])
-        self.assertEqual(repo.checksum_type, 'sha256')
+        for checksum_type in 'sha1', 'sha256':
+            for downloadpolicy in 'background', 'immediate':
```

Comment:
same redundancy here for at least both download policies

Reply:
agree with same